### ISSUE TITLE
Mods Looking for Support

### ISSUE THREAD

**Issue by WaitingIdly** [roidrole/A-Dark-Mode#10]
identified a large number of GUIs that are not currently supported by this texture pack. each link either goes directly to the texture looking for support (if there is only one) or the folder in which the relevant textures are stored for the 1.12.2 version of the mod, to the best of my knowledge.
if/when i have time, ill look into converting some of these. not very good with textures, but a simple color swap cant be that hard. right? 
\* = indicates that the mod is part of Divine Journey 2, which is where i identified most of these.
^ = indicates the the mod is part of Enigmatica 2 Expert Extended (figured it was worth noting). (just edited this in)

### These mods don't have support:

- [x] ^[PackagedExCrafting](https://github.com/TheLMiffy1111/PackagedExCrafting/tree/1.12/src/main/resources/assets/packagedexcrafting/textures/gui)
- [x] ^potentially [PackagedAstral](https://github.com/TheLMiffy1111/PackagedAstral/tree/1.12/src/main/resources/assets/packagedastral/textures/gui), although it has the same textures as Astral Sorcery.
- [x] ^*[Astral Sorcery](https://github.com/HellFirePvP/AstralSorcery/tree/1.12.2/src/main/resources/assets/astralsorcery/textures/gui) - has a custom background texture based on Marble, so it may not be reasonable to have a dark mode for its GUIs.
- [x] [PackagedThaumic](https://github.com/TheLMiffy1111/PackagedThaumic/tree/1.12/src/main/resources/assets/packagedthaumic/textures/gui)
- [x] ^[PackagedDraconic](https://github.com/TheLMiffy1111/PackagedDraconic/tree/1.12/src/main/resources/assets/packageddraconic/textures/gui)
- [x] [PackagedAvaritia](https://github.com/TheLMiffy1111/PackagedAvaritia/tree/1.12/src/main/resources/assets/packagedavaritia/textures/gui)
- [x] [Embers](https://github.com/MysticMods/Embers/tree/release/1.12.2/src/main/resources/assets/embers/textures/gui)
- [x] *[Roots](https://github.com/MysticMods/Roots/tree/1.12.2/src/main/resources/assets/roots/textures/gui)
- [x] [Extreme Sound Muffler: Legacy](https://github.com/Lyfts/ExtremeSoundMuffler-Legacy/tree/1.12.2/src/main/resources/assets/extremesoundmuffler/textures/gui) is particularly obtrusive due to adding a button directly to the vanilla inventory.
- [x] [Calculator](https://github.com/SonarSonic/Calculator/tree/1.12.2/src/main/resources/assets/calculator/textures/gui)
- [x] [Aetherworks](https://github.com/V0idWa1k3r/Aetherworks/tree/master/src/main/resources/assets/aetherworks/textures/compat/jei)
- [x] [Soot](https://github.com/DaedalusGame/Soot/tree/master/src/main/resources/assets/soot/textures/gui)
- [x] *[Alchemistry](https://github.com/SmashingMods/Alchemistry/tree/1.12.2/src/main/resources/assets/alchemistry/textures/gui/container)
- [x] [Dynamistics](https://github.com/eutro/dynamistics/tree/master/src/main/resources/assets/dynamistics/textures/gui)
- [x] *[EvilCraft](https://github.com/CyclopsMC/EvilCraft/tree/release-1.12/src/main/resources/assets/evilcraft/textures/gui)
- [x] [Multiblock'd](https://github.com/CleanroomMC/Multiblocked/blob/master/src/main/resources/assets/multiblocked/textures/gui)
- [x] [Arcane Archives](https://github.com/AranaiRa/ArcaneArchives/tree/master/src/main/resources/assets/arcanearchives/textures/gui)
- [x] Woot for specifically [Woot Stygian Iron Anvil JEI Item Slots](https://github.com/Ipsis/Woot/blob/1_12/src/main/resources/assets/woot/gui/jei/stygiananvil.png) and [Woot Heart Background](https://github.com/Ipsis/Woot/blob/1_12/src/main/resources/assets/woot/textures/gui/heart.png)
- [x] ^*[Thaumic Augmentation](https://github.com/TheCodex6824/ThaumicAugmentation/tree/master/src/main/resources/assets/thaumicaugmentation/textures/gui)
- [x] *[Demagnetize](https://github.com/comp500/Demagnetize/blob/1.12/src/main/resources/assets/demagnetize/textures/gui/demagnetizer.png)
- [x] *[Draconic Additions](https://github.com/FoxMcloud5655/Draconic-Additions/tree/1.12.2/src/main/resources/assets/draconicadditions/textures/gui)
- [x] *[Simply Jetpacks 2](https://github.com/Tomson124/SimplyJetpacks2/tree/1.12/src/main/resources/assets/simplyjetpacks/textures/gui)
- [x] *[Better P2P](https://github.com/AE2-UEL/BetterP2P/blob/main/src/main/resources/assets/betterp2p/textures/gui/advanced_memory_card.png)
- [x] [Blood Arsenal](https://github.com/Arcaratus/BloodArsenal/tree/1.12/src/main/resources/assets/bloodarsenal/textures/gui)
- [x] *[Atum 2](https://github.com/TeamMetallurgy/Atum2/tree/1.12.2-release/src/main/resources/assets/atum/textures/gui)
- [x] *[Galacticraft Core](https://github.com/TeamGalacticraft/Galacticraft-Legacy/tree/master-1.12/src/main/resources/assets/galacticraftcore/textures/gui) (via Galacticraft Legacy)
- [x] *[Galacticraft Planets](https://github.com/TeamGalacticraft/Galacticraft-Legacy/tree/master-1.12/src/main/resources/assets/galacticraftplanets/textures/gui) (via Galacticraft Legacy)
- [x] *[Planet Progression](https://github.com/MJRLegends/Planet-Progression/tree/master/src/main/resources/assets/planetprogression/textures/gui)
- [x] *[AbyssalCraft](https://github.com/Shinoow/AbyssalCraft/tree/master/src/main/resources/assets/abyssalcraft/textures/gui)
- [x] *[Just Enough Reactors](https://github.com/AuraDevelopmentTeam/JustEnoughReactors/blob/master/src/main/resources/assets/justenoughreactors/textures/gui/background.png)
- [x] *[Bewitchment](https://github.com/Um-Mitternacht/Bewitchment/tree/7e3abee418a0a14bf9a40464fcc581996bc97004/src/main/resources/assets/bewitchment/textures/gui)
- [x] *[Aether Legacy](https://github.com/The-Aether-Team/The-Aether-Archived/tree/1.12.2/src/main/resources/assets/aether_legacy/textures/gui)
- [x] *[LightningCraft](https://github.com/sblectric/LightningCraft/tree/master/src/main/resources/assets/lightningcraft/textures/gui)
- [x] *[AgriCraft](https://github.com/AgriCraft/AgriCraft/tree/1.12.2/src/main/resources/assets/agricraft/textures/gui)
- [x] *[WirelessRedstone-CBE](https://github.com/TheCBProject/WirelessRedstone/tree/master/src/main/resources/assets/wrcbe/textures/gui)
- [x] *[Solar Flux Reborn](https://github.com/dragon-forge/SolarFluxReborn/tree/1.12.2/src/main/resources/assets/solarflux/textures/gui)
- [x] ^*[PlusTiC](https://github.com/TeamDman/PlusTiC/tree/1.12/src/main/resources/assets/plustic/textures/gui)
- [x] ^*[Integrated NBT](https://github.com/SCLeoX/IntegratedNBT/tree/master/src/main/resources/assets/integratednbt/textures/gui) most the other Integrated [X] mods have comparability.
- [x] *[Baubley Heart Canisters](https://github.com/Traverse-Joe/Baubley-Heart-Canisters/blob/37a74576e3392ea8565a44a63b2b1358c226ef8b/src/main/resources/assets/bhc/textures/gui/heart_pendant.png)
- [x] ^*[Thaumic Energistics Extended Life](https://github.com/Delfayne/ThaumicEnergistics/tree/main/src/main/resources/assets/thaumicenergistics/textures/gui)

### These specific GUIs don't have support:
i noticed a few GUIs from mods that already have compatibility that were missing dark mode, mainly caused by forks of the mod.

- [x] ^*[Applied Energistics 2 Extended Life CPU](https://github.com/AE2-UEL/Applied-Energistics-2/blob/master/src/main/resources/assets/appliedenergistics2/textures/guis/cpu_selector.png) 
- [x] ^*[Applied Energistics 2 Extended Life Quantum Bridge](https://github.com/AE2-UEL/Applied-Energistics-2/blob/master/src/main/resources/assets/appliedenergistics2/textures/guis/qbgui.png)
- [x] *[Modular Machinery](https://github.com/HellFirePvP/ModularMachinery/tree/1.12/src/main/resources/assets/modularmachinery/textures/gui) non-CE has a number of texture issues which appear to be because of CE support. not sure how viable having compat with both is currently - might want to ask CE to change the name of their texture files? otherwise the only way to have compat is to make a separate texture pack specifically for CE (or for non-CE, just need to be separate).

### A Rather Annoying Texture:

- [x] *Erebus Offering Altar in JEI - added by MoreTweaker for JEI, under `assets/moretweaker/textures/gui/erebus_offering.png`. since the source code isnt public (even though its licensed MIT), it cant be linked - heres the raw texture.
    ![erebus_offering](https://github.com/user-attachments/assets/570e4042-9bc9-41dd-8edd-09003eb19de4)

ran into this project via E2EE.

**Comment by roidrole** [comment]
> *[Modular Machinery](https://github.com/HellFirePvP/ModularMachinery/tree/1.12/src/main/resources/assets/modularmachinery/textures/gui) non-CE has a number of texture issues which appear to be because of CE support. not sure how viable having compat with both is currently - might want to ask CE to change the name of their texture files? otherwise the only way to have compat is to make a separate texture pack specifically for CE (or for non-CE, just need to be separate).

Are these texture issues default or weirdly? It would help me much if you could provide an example screenshot.
In the meantime, I'll fix (in 1.10.0) the guiblueprint as that's a texture not present in CE.

**Comment by roidrole** [comment]
I'll leave this thread open for a bit because, let's be honest, I probably missed some textures here and there, but they should all be added in version 1.10

**Comment by WaitingIdly** [comment]
heres an example of Modular Machinery textures behaving incorrect in JEI (once again, in DJ2)
![image](https://github.com/user-attachments/assets/72b8931c-0790-4041-81d1-dfbba10ae588)
the way fluids are represented have changed from a vertical column into a square, and the entire texture atlas is slightly misaligned.

this is the relevant file: [Modular Machinery](https://github.com/HellFirePvP/ModularMachinery/blob/1.12/src/main/resources/assets/modularmachinery/textures/gui/jeirecipeicons.png) vs [Modular Machinery Community Edition](https://github.com/NovaEngineering-Source/ModularMachinery-Community-Edition/blob/master/src/main/resources/assets/modularmachinery/textures/gui/jeirecipeicons.png). the best way to resolve this might be by asking MMCE to rename the file?

**Comment by WaitingIdly** [comment]
thanks a lot for taking the time to complete all of these. its really impressive. 
while checking out the textures in v10.0, i noticed a few things missing or not working properly.

## i missed a few textures:
- [ ] AbyssalCraft is being reworked into v2, and has an actively developing alpha version for this as its main repo. i linked to the AbyssalCraft 2 files, when i should have linked to the AbyssalCraft 1 files, as those are what are currently used. The [Coin Engraver](https://github.com/Shinoow/AbyssalCraft/blob/1.12.2-1.x.x/src/main/resources/assets/abyssalcraft/textures/gui/container/engraver.png), its [JEI version](https://github.com/Shinoow/AbyssalCraft/blob/1.12.2-1.x.x/src/main/resources/assets/abyssalcraft/textures/gui/container/engraver_nei.png), and the [Transmutator](https://github.com/Shinoow/AbyssalCraft/blob/1.12.2-1.x.x/src/main/resources/assets/abyssalcraft/textures/gui/container/transmutator.png) are the textures i missed.
- [ ] Custom Main Menu has a single texture to [refresh](https://github.com/lumien231/Custom-Main-Menu/blob/master/src/main/resources/assets/custommainmenu/textures/gui/buttons.png) the background.
- [ ] Embers Unofficial Extended Life is a fork of Embers which adds a few textures - the [Geologic Separator](https://github.com/Ender-Development/Embers-Extended-Life/blob/rekindled/src/main/resources/assets/embers/textures/gui/jei_geologic_separator.png), [Dawnstone Anvil](https://github.com/Ender-Development/Embers-Extended-Life/blob/rekindled/src/main/resources/assets/embers/textures/gui/jei_dawnstone_anvil.png), [Boiler/Steam Engine/Reaction Chamber](https://github.com/Ender-Development/Embers-Extended-Life/blob/rekindled/src/main/resources/assets/embers/textures/gui/jei_boiler.png) (all use the same texture for JEI). 
- [ ] [Neeve's Applied Energistics 2 Extended Life Additions](https://github.com/AE2-UEL/NAE2/tree/master/src/main/resources/assets/nae2/textures/gui) has a few textures, just completely missed this mod.

## issues with currently existing compat (most are links to png files in this repository):

<details><summary>Bewitchment</summary>

- [ ] Bewitchment [Brew](https://github.com/roidrole/A-Dark-Mode-Addon/blob/main/Dark%20Mode-Addon/assets/bewitchment/textures/gui/jei_brew.png), [Brazier](https://github.com/roidrole/A-Dark-Mode-Addon/blob/main/Dark%20Mode-Addon/assets/bewitchment/textures/gui/jei_brazier.png), [Cauldron](https://github.com/roidrole/A-Dark-Mode-Addon/blob/main/Dark%20Mode-Addon/assets/bewitchment/textures/gui/jei_cauldron.png), and [Frostfire](https://github.com/Um-Mitternacht/Bewitchment/blob/7e3abee418a0a14bf9a40464fcc581996bc97004/src/main/resources/assets/bewitchment/textures/gui/jei_frostfire.png) didnt get converted (the first three are textures in the texture pack, Frostfire isnt).
</details> 

<details><summary>Extended Crafting Ender Crafting</summary>

- [ ] Extended Crafting Ender Crafting appears oddly in [JEI](https://github.com/roidrole/A-Dark-Mode-Addon/blob/main/Dark%20Mode-Addon/assets/extendedcrafting/textures/jei/ender_crafting.png) with its light arrow texture. the background also appears odd, and may want to be changed so its not such a sharp border?
![image](https://github.com/user-attachments/assets/9bfded4a-5eaa-4d1d-a682-b41d5a088071).
</details> 

<details><summary>Extended Crafting Quantum Compressor GUI</summary>

- [ ] Extended Crafting [Quantum Compressor GUI](https://github.com/roidrole/A-Dark-Mode-Addon/blob/main/Dark%20Mode-Addon/assets/extendedcrafting/textures/gui/compressor.png) (only the GUI, JEI version doesnt have this issue) is lighter colored than the rest of the Extended Crafting GUIs and is missing the arrow
![image](https://github.com/user-attachments/assets/49eaa16b-a584-4fa1-bc0e-0467a0e7949f)
![image](https://github.com/user-attachments/assets/4dd219c1-0f3e-495f-bd18-20d9ddc5f547)
</details> 

<details><summary>Roots Loot Category</summary>

- [ ] Roots [Loot Category in JEI](https://github.com/roidrole/A-Dark-Mode-Addon/blob/main/Dark%20Mode-Addon/assets/roots/textures/gui/jei/loot.png) did not get converted 
</details> 

## some issues that i do not believe are fixable as a pure texture pack:

<details><summary>Embers Unofficial Extended Life</summary>

- [ ] similar to the MM/MMCE issue, when using the fork of Embers, Embers Unofficial Extended Life, the [Alchemy JEI](https://github.com/Ender-Development/Embers-Extended-Life/blob/rekindled/src/main/resources/assets/embers/textures/gui/jei_alchemy.png) does not appear properly. 
![image](https://github.com/user-attachments/assets/e76be15f-5e4e-4648-bc03-6a145dc7d15f)
</details> 

<details><summary>PackagedAuto JEI</summary>

- [ ] [PackagedAuto JEI](https://github.com/TheLMiffy1111/PackagedAuto/blob/1.12/src/main/java/thelm/packagedauto/integration/jei/category/PackageRecipeCategory.java#L70) has a visual bug which is set directly via Java.
![image](https://github.com/user-attachments/assets/c72c1a63-0f44-4bab-9176-8b5f1fce4d04)
</details>

**Comment by roidrole** [comment]
>Extended Crafting Ender Crafting appears oddly in [JEI](https://github.com/roidrole/A-Dark-Mode-Addon/blob/main/Dark%20Mode-Addon/assets/extendedcrafting/textures/jei/ender_crafting.png) with its light arrow texture. the background also appears odd, and may want to be changed so its not such a sharp border?

The arrow I can fix no problem. However, I struggle to understand what you mean by « not such a sharp border ». I’ll check the textures later, but I believe the gray border is added by JEI itself (maybe extended crafting, but shared across all guis). Are you suggesting I do a grey -> green fade in the background? I guess I could also make the entire background gray, but I want to, where possible, keep the original mod’s feel

**Comment by roidrole** [comment]
Created an [issue](https://github.com/Ender-Development/Embers-Extended-Life/issues/46) on Ember's github

**Comment by WaitingIdly** [comment]
when i say "not such a sharp border" i am referring to the sharp line between the green and the gray, yes. checking without this texture pack reveals that this is not a problem on your end, but by Extended Crafting.

**Comment by roidrole** [comment]
As all requested mod support has been added and the last issues fixed (I was mainly waiting on a MM:CE release), I will consider this issue closed. PackagedAuto JEI's issue probably won't get fixed. Feel free to make another issue if you wish to request more mods!
